Sources/Kafka/RDKafka/RDKafkaStatistics.swift

```diff
@@ -0,0 +1,184 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-kafka-gsoc open source project
+//
+// Copyright (c) 2023 Apple Inc. and the swift-kafka-gsoc project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-kafka-gsoc project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+// This file was generated from JSON Schema using quicktype, do not modify it directly.
+// To parse the JSON, add this file to your project and do:
+//
+//   let statistics = try? newJSONDecoder().decode(RDKafkaStatistics.self, from: jsonData)
+
+// MARK: - Statistics
+
+struct RDKafkaStatistics: Hashable, Codable {
+    let name, clientID, type: String?
+    let timestamp, time, age, replyQueue: Int?
+    let messageCount, messageSize, messageMax, messageSizeMax: Int?
+    let simpleCnt, metadataCacheCount: Int?
+//    let brokers: [String: Broker]?
+//    let topics: [String: Topic]?
+//    let cgrp: Cgrp?
```

Comment:
Can we remove the commented out code please

Reply:
Let's remove it for now and add back when will add those metrics